package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@kaliber/routing",
+  "version": "0.0.0",
+  "main": "index.js",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "type": "module",
+  "keywords": [
+    "react",
+    "routing"
+  ],
+  "files": [
+    "src/**",
+    "LICENSE",
+    "readme.md",
+    "package.json"
+  ],
+  "private": false,
+  "license": "MIT",
+  "description": "Provides routing for React",
+  "repository": "https://github.com/kaliberjs/routing.git",
+  "contributors": [
+    "Erik Westra <erik@kaliber.net>"
+  ],
+  "scripts": {
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest"
```

Comment:
Yeah JEST!

Ik heb laatst deze cursus afgerond: https://testingjavascript.com/

Reply:
Als je het leuk vindt: uiteindelijk wil ik de React functies in `routing.js` ook nog testen. Ik had alleen niet de puf om dat nog toe te voegen.